packages/design-system/src/components/RichRadioButton/RichRadioButton.types.tsx

```diff
@@ -0,0 +1,25 @@
+import { IconNameWithSize } from '@talend/icons';
+import { ReactElement } from 'react';
+import { TagVariant } from '../Tag/Tag';
+
+export interface RichRadioButtonTag {
+	name: string;
+	variant?: keyof typeof TagVariant;
+}
+
+export interface RichRadioButtonProps {
+	dataFeature?: string;
+	description?: string;
+	asset?: {
+		illustration?: ReactElement;
+		name?: IconNameWithSize<'L'>;
```

Comment:
I think this type is too restrictive cause we would like to be able to provide remote url.